rule/receiver-naming.go

```diff
@@ -66,6 +107,17 @@ func (w lintReceiverName) Visit(n ast.Node) ast.Visitor {
 		})
 		return w
 	}
+
+	if w.receiverNameMaxLength > 0 && len(name) > w.receiverNameMaxLength {
```

Comment:
I thought it would be better to use rune.
```suggestion
	if w.receiverNameMaxLength > 0 && len([]rune(name)) > w.receiverNameMaxLength {
```